openapi/mx_platform_api.yml

```diff
@@ -791,6 +791,11 @@ components:
           example: chase
           nullable: true
           type: string
+        instructional_text:
+          example: Some instructional text <a href="https://example.url.chase.com/instructions">for
```

Comment:
Can you add an attribute to that `<a />` tag? `id="instructional_text`. This `id` is required and will be provided on all links inside of `instructional_text`.

Reply:
Yup!

Reply:
Done!